src/reanimated2/background.ts

```diff
@@ -0,0 +1,62 @@
+'use strict';
+import NativeReanimatedModule from './NativeReanimated';
+import { shouldBeUseWeb } from './PlatformChecker';
+import type { WorkletFunction } from './commonTypes';
+import type { WorkletRuntime } from './runtimes';
+import {
+  makeShareableCloneOnUIRecursive,
+  makeShareableCloneRecursive,
+} from './shareables';
+
+const SHOULD_BE_USE_WEB = shouldBeUseWeb();
+
+export type BackgroundQueue = {
```

Comment:
So, from my understanding this type is just to signify that it's a HostObject and nothing should be accessed on it. How about this pattern then?
![BackgroundQueue](https://github.com/software-mansion/react-native-reanimated/assets/40713406/2e65ba8d-0235-447b-be8f-e10c47ccc247)

Reply:
Done in 74529a65b0c3e52bf5da71f653c83bf5a16b084f